src/NServiceBus.Transport.RabbitMQ/Administration/ManagementClient/ManagementClientFactory.cs

```diff
@@ -0,0 +1,8 @@
+﻿#nullable enable
+
+namespace NServiceBus.Transport.RabbitMQ.Administration.ManagementClient;
+
+class ManagementClientFactory(ConnectionConfiguration connectionConfiguration) : IManagementClientFactory
```

Comment:
Why do we need this factory class at all? Seems like we could remove and just create the management client directly?